lib/ex_vatcheck/vies_client.ex

```diff
@@ -7,6 +7,11 @@ defmodule ExVatcheck.VIESClient do
   alias ExVatcheck.VIESClient.XMLParser
 
   @wsdl_url "https://ec.europa.eu/taxation_customs/vies/checkVatService.wsdl"
+  if System.otp_release() == "23" do
```

Comment:
this will make OTP 24 not use tls1.3 which doesn't seem right. I'll update this to use a `Version.compare`